tests/impls/vis_sal_classifier/test_occlusion_classifier.py

```diff
@@ -0,0 +1,101 @@
+from unittest import TestCase
+
+import numpy as np
+
+from xaitk_saliency.impls.vis_sal_classifier.occlusion_scoring import OcclusionScoring
+from xaitk_saliency import ImageClassifierSaliencyMapGenerator
+
+
+class TestOcclusionBasedScoring (TestCase):
+
+    def test_init_(self) -> None:
+        """
+        Test if implementation is usable.
+        """
+        impl = OcclusionScoring()
+        assert impl.is_usable() and isinstance(impl, ImageClassifierSaliencyMapGenerator)
+
+# Below test case does pass but we should have more discussion
```

Comment:
This is testing multiple images as input to the function. This addition to the implementation brings in an additional dimension to all input and output args.  @Purg Do you think this is something in line with our plan or something we should discuss more?

Reply:
This is currently not represented in the API, so this test would not apply to this current version. I believe we discussed it once but decided to go with the simpler version to start as a balance between imposed complexity between user and implementer. We can revisit this later if usage reveals more advantage to multi-image vectorization.